drv/Maps.drv

```diff
@@ -403,6 +494,110 @@ public class MAPS {
 
 		@Override
 		public VALUE_COLLECTION VALUE_GENERIC values() { if (values == null) return VALUE_COLLECTIONS.unmodifiable(map.values()); return values; }
+
+		@Override
+		public boolean isEmpty() { return map.isEmpty(); }
+
+		@Override
+		public int hashCode() { return map.hashCode(); }
+
+		@Override
+		public boolean equals(final Object o) {
+			if (o == this) return true;
+			if (o instanceof UnmodifiableMap) return map.equals(((UnmodifiableMap) o).map);
```

Comment:
Not dangerous, but frankly unnecessary and possibly dangerous (e.g., the unmodifiable wrapper could do things you wanna do when you access the map).